modules/install-creating-install-config-aws-local-zones.adoc

```diff
@@ -3,33 +3,31 @@
 
 :_content-type: PROCEDURE
 [id="install-creating-install-config-aws-local-zones_{context}"]
-= Modifying an installation configuration file to use AWS Local Zones subnets
+= Modifying an installation configuration file to use AWS Local Zones
 
-Modify an `install-config.yaml` file to include AWS Local Zones subnets.
+Modify an `install-config.yaml` file to include AWS Local Zones.
 
 .Prerequisites
 
-* You created subnets by using the procedure "Creating a subnet in AWS Local Zones".
 * You created an `install-config.yaml` file by using the procedure "Creating the installation configuration file".
 
 .Procedure
 
-* Add the VPC and Local Zone subnets as the values of the `platform.aws.subnets` property. As an example:
+* Add Local Zones as values to the `platform.aws.szones` property. As an example:
```

Comment:
1) incorrect config path

```suggestion
* Add Local Zones as values to the `compute[?name==edge].platform.aws.zones` property. As an example:
```

2) this is not an example. I think examples must have real values. This is an example (provided in the section before): https://github.com/openshift/openshift-docs/pull/60771/files#diff-daa4d0981964bb25837f20e7d10e26e914dddd4a1eea03d45013cf43617879b2R34

3) Maybe  this section is duplicated with before, or need to be reviewed preventing duplicated, and presenting examples before explaining the config.

Reply:
Updated lead-in sentence to:

> Add Local Zones as values to the `platform.aws.zones` property in the `compute.name.edge` stanza of the configuration file. For example

According to style guide requirements, with should be using user-replaced values (for example, <local_zone_name_1>). I updated the examples by providing real Local Zone names in the Installing a cluster to quickly extend workers to edge compute pools section. WDYT?

If OK with this, I can apply throughout the doc set.

Reply:
We need to revisit this entire section closely (Install quickly). Maybe copying the content to Google docs will be more productive. WDYT?

Reply:
@dfitzmau I only found the TOC, not the content. I just added a proposal for this section here: ["Proposal 4.14 (quickly install)"](https://docs.google.com/document/d/1cBPJiPj4v11Vr6tP_SPCy0pFQ9ibamzSIz7YRUyKQWI/edit#heading=h.h6gmc29s48jd)

It is adjusting the content to the chronological approach I mentioned in some of my comments. This section is the one added in the 4.14.

Reply:
Hi @mtulio . Thanks for the suggested content in the doc. Do you want me to remove most of the content stated in https://60771--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-localzone.html#installation-cluster-quickly-extend-workers_installing-aws-localzone ?

Reply:
> Hi @mtulio . Thanks for the suggested content in the doc. Do you want me to remove most of the content stated in https://60771--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-localzone.html#installation-cluster-quickly-extend-workers_installing-aws-localzone ?

Yes, the proposal is to replace the entire section "Installing a cluster quickly in AWS Local Zones".